enterprise-suite/console_test/console_test.go

```diff
@@ -0,0 +1,98 @@
+package console
+
+import (
+	"fmt"
+	"testing"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/lightbend/console_test/args"
+
+	"github.com/lightbend/console_test/util/helm"
+	"github.com/lightbend/console_test/util/lbc"
+	"github.com/lightbend/console_test/util/minikube"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// The following variables are used in this package to access kubernetes
+// and make requests on Console components.
+var k8sClient *kubernetes.Clientset
+var consoleAddr string
+var prometheusAddr string
+var monitorAPIAddr string
+
+var _ = BeforeSuite(func() {
```

Comment:
I think instead of having a global BeforeSuite, it's better to extract this into a function, and call it from the BeforeSuite of the various `Describe`. That lets us customize as needed in those tests. It's also a little more explicit (it's not obvious the beforesuite is executing from another file).

Reply:
Right, this whole restructuring into packages needed that as well. Now environment initialization is in `testenv` package.